tests/FSharp.Compiler.ComponentTests/xunit.runner.json

```diff
@@ -1,7 +1,4 @@
 ﻿{
-    "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
-    "appDomain": "ifAvailable",
-    "shadowCopy": false,
-    "parallelizeTestCollections": false,
-    "maxParallelThreads": 1
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+    "shadowCopy": false
```

Comment:
Setting this to default true would probably be even more demanding for our tests now, right?

Reply:
Yes, but we kind of want to parallelize this one and FSharpSuite at least. This will require some finetuning, for sure.

Reply:
Sure. Note that _some_ of the component tests are inherently supposed to run sequentially - they read BCL and use memory-mapped files, hence forcing things work in memory will increase the memory consumption of the compiler itself which we don't want.

That was the idea of the suite but then a bunch of things crept in and so this doesn't apply to the whole project anymore, but still might be useful for you to know :)